Gruntfile.js

```diff
@@ -197,7 +193,7 @@ module.exports = function( grunt ) {
 					'!<%= dirs.js %>/admin/*.min.js',
 					'!<%= dirs.js %>/frontend/*.min.js'
 				],
-				tasks: ['jshint', 'uglify']
+				tasks: ['uglify']
```

Comment:
No more lint while converting files? Maybe allow only warnings?

Reply:
*grumble grumble* I fixed all of the ESLint errors 😫